Gemfile

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+gem 'danger'
+gem 'danger-android_lint'
+gem 'danger-commit_lint'
+gem 'danger-kotlin_detekt'
+gem 'danger-shroud'
```

Comment:
minor: missing end of file break line; you may wait for the others to commit changes if they raise anything else.